src/opm/parser/eclipse/EclipseState/Schedule/Well/WellTestState.cpp

```diff
@@ -19,24 +19,27 @@
 #include <stdexcept>
 #include <algorithm>
 
+#include <cassert>
 #include <opm/parser/eclipse/EclipseState/Schedule/Well/WellTestConfig.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/Well/WellTestState.hpp>
 
 namespace Opm {
 
-    void WellTestState::addClosedWell(const std::string& well_name, WellTestConfig::Reason reason, double sim_time) {
-        if (this->hasWell(well_name, reason))
-            return;
-
-        this->wells.push_back({well_name, reason, sim_time, 0});
+    void WellTestState::closeWell(const std::string& well_name, WellTestConfig::Reason reason, double sim_time) {
+        if (this->hasWell(well_name, reason)) {
```

Comment:
Please remove the superfluous braces

Reply:
I do not agree with this while I will do it to address the comment.

Reply:
Hehe - I was actually not expecting you to do it; thank you for willingness lubricate the review process - just leave the braces if you prefer!